cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -528,6 +528,57 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     clusterList.waitForPage();
   });
 
+  describe('Cluster Details Tabs', () => {
+    beforeEach( () => {
+      HomePagePo.goTo();
+      const burgerMenu = new BurgerMenuPo();
+
+      BurgerMenuPo.toggle();
+      const clusterManagementNavItem = burgerMenu.links().contains(`Cluster Management`);
+
+      clusterManagementNavItem.should('exist');
+      clusterManagementNavItem.click();
```

Comment:
lets avoid using `goTo()` and use `navTo()` instead. we can use `ClusterManagerListPagePo.navTo();` here.